### PR TITLE
Variables update and minor bug fixes

### PR DESCRIPTION
**Pull request by danishjamal104**
Fixes: https://fynarfin.atlassian.net/browse/PHEE-77?atlOrigin=eyJpIjoiODY1NGZiNmM1Nzk4NGQxNzg0OGFmMzcyYTM1ZmEwMTkiLCJwIjoiaiJ9

### CODE REVIEW

**Comment by danishjamal104** [comment]
@avikganguly01 and @SubhamPramanik please review and merge. Thanks :)

File: src/main/java/hu/dpc/phee/operator/importer/RecordParser.java [hunk 1]
```diff
@@ -201,7 +201,7 @@ public void processTask(DocumentContext json) {
             return;
         }
 
-        Long workflowInstanceKey = json.read("$.value.workflowInstanceKey");
+        Long workflowInstanceKey = json.read("$.value.processInstanceKey");
```
Comment (avikganguly01) [inline src/main/java/hu/dpc/phee/operator/importer/RecordParser.java:204]: Shouldn't this be process definition key and not process instance id?

**Review by avikganguly01 — CHANGES_REQUESTED** [review]

File: src/main/java/hu/dpc/phee/operator/importer/RecordParser.java (hunk 1 above)
Comment (danishjamal104) [reply src/main/java/hu/dpc/phee/operator/importer/RecordParser.java:204]: Below are the changes
`workflowKey` : `processDefinitionKey`
`workflowInstanceKey `  : `processInstanceKey `

**Review by avikganguly01 — CHANGES_REQUESTED** [review]

File: src/main/java/hu/dpc/phee/operator/importer/RecordParser.java [hunk 2]
```diff
@@ -80,7 +80,7 @@ public void addVariableToEntity(DocumentContext newVariable, String bpmnProcessI
         }
 
         String name = newVariable.read("$.value.name");
-        Long workflowInstanceKey = newVariable.read("$.value.workflowInstanceKey");
+        Long workflowInstanceKey = newVariable.read("$.value.processDefinitionKey");
```
[the remark this answers is not in the document]
Comment (danishjamal104) [reply src/main/java/hu/dpc/phee/operator/importer/RecordParser.java]: fixed

**Review by avikganguly01 — APPROVED** [review]

**Pull request merged**